### DISCUSSION TITLE
how to configure dynamic routing

### DISCUSSION THREAD

**Discussion by Jixiangup** [apache/apisix#6045] [Q&A]
I want to configure my parameters in path, how should I configure him? I have configured it now but `apisix` gives me `404 responses`

- routing config

![image](https://user-images.githubusercontent.com/75556346/148527130-56eb34d6-2eed-4630-8c8f-fc204acd8fdc.png)

- apisix 404 response

![image](https://user-images.githubusercontent.com/75556346/148527024-7f96d2b6-0dc6-462b-9035-7405770e4fa9.png)

**Comment by juzhiyuan** [comment]
Hi @bnyte, could you please paste the complete Route configuration?
